src/main/resources/reference.conf

```diff
@@ -12,9 +12,13 @@ billow {
         # How many retries to make for failing requests (uses AWS exponential backoff)
         maxErrorRetry = 10
 
-        # AWS account ID
-        accountNumber = 123456789012
 
+
+        # AWS account
+        # The account and Ids are used for local development and testing and should be
+        # commented out when deployed inside airbnb.
+
+        # accountNumber = 123456789012
```

Comment:
How about...

```

# The AWS Account Number and Access Key fields are commented out because we
# prefer the use of IAM Roles in productions. These fields are useful for
# local development.

# AWS Account Number
# accountNumber = 123456789012

# AWS Access Key
# accessKeyId = AK47
# secretKeyId = OMGsoSecret
```